package.json

```diff
@@ -29,15 +28,17 @@
   "devDependencies": {
     "@nextcloud/axios": "^2.4.0",
     "@nextcloud/babel-config": "^1.0.0",
-    "@nextcloud/browserslist-config": "^2.3.0",
-    "@nextcloud/eslint-config": "^8.2.1",
+    "@nextcloud/browserslist-config": "^3.0.0",
+    "@nextcloud/dialogs": "^5.1.2",
+    "@nextcloud/eslint-config": "^8.3.0",
     "@nextcloud/initial-state": "^2.1.0",
+    "@nextcloud/l10n": "^2.2.0",
     "@nextcloud/router": "^2.2.0",
     "@nextcloud/stylelint-config": "^2.3.1",
-    "@nextcloud/vue": "^7.12.4",
+    "@nextcloud/vue": "^8.8.1",
     "@nextcloud/webpack-vue-config": "^6.0.0",
     "vue": "^2.7.15",
-    "vue-material-design-icons": "^5.2.0"
+    "vue-material-design-icons": "^5.3.0"
```

Comment:
Just in case, you didn't need to run `npm i --save-dev vue-material-design-icons@^5.3.0` since there was no major version bump for this pkg. `npm update` would have updated this package without touching to `package.json`.